gradle/dependency-locks/apiDependenciesMetadata.lockfile

```diff
@@ -2,8 +2,9 @@
 # Manual edits can break the build and are not advised.
 # This file is expected to be part of source control.
 com.atlassian.performance.tools:concurrency:1.1.0
-com.atlassian.performance.tools:jira-actions:3.2.0
-com.atlassian.performance.tools:jvm-tasks:1.0.1
+com.atlassian.performance.tools:jira-actions:3.13.3
```

Comment:
These got bumped, because I did a blanked bump of all libs via `./gradlew :dependencies --write-locks`. Bumping them selectively was too much work, too many conflicts to attack one by one.